src/platform/terminal/vscode/terminalServiceImpl.ts

```diff
@@ -75,6 +75,13 @@ export class TerminalServiceImpl extends Disposable implements ITerminalService
 	get onDidWriteTerminalData(): Event<TerminalDataWriteEvent> {
 		return window.onDidWriteTerminalData;
 	}
+	get onDidChangeTerminalState(): Event<Terminal> {
+		return window.onDidChangeTerminalState;
+	}
+
+	get onDidChangeTerminalShellType(): Event<string> {
+		return onDidChangeShellType;
+	}
```

Comment:
This isn't used AFAICT so we shouldn't have it. Most events/properties here just forward the extension api stuff over.